### PR TITLE
Update enable-tls-between-components and add TLS rotation description

### PR DESCRIPTION
**Pull request by lulukelu**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [x] Delete files
- [x] Change aliases
- [x] Need modification after applied to another branch 
- [x] Might cause conflicts after applied to another branch

### CODE REVIEW

File: enable-tls-between-components.md [hunk 1]
```diff
@@ -189,7 +189,13 @@ aliases: ['/docs-cn/dev/enable-tls-between-components/','/docs-cn/dev/how-to/sec
 
 ## 证书重加载
 
-TiDB、PD 和 TiKV 和各种 Client 都会在每次新建相互通讯的连接时重新读取当前的证书和密钥文件内容，实现证书和密钥的重加载。目前暂不支持 CA 的重加载。
+目前支持TiDB、TiKV、PD的TLS证书，部署数据中心和Cloud上时，自动轮换TLS证书，无需重启TiDB集群；暂不支持TiFlash，TiCDC的自动轮换TLS证书。
```
Comment (qiancai) [inline enable-tls-between-components.md:192]:
```suggestion
目前支持 TiDB、TiKV、PD 的 TLS 证书，部署数据中心和 Cloud 时，支持自动轮换 TLS 证书，无需重启 TiDB 集群。 TiFlash 和 TiCDC 暂不支持自动轮换 TLS 证书。
```

File: enable-tls-between-components.md [hunk 2]
```diff
@@ -189,7 +189,13 @@ aliases: ['/docs-cn/dev/enable-tls-between-components/','/docs-cn/dev/how-to/sec
...
+
+TiDB、PD 和 TiKV 和各种 Client 都会在每次新建相互通讯的连接时重新读取当前的证书和密钥文件内容，实现证书和密钥的重加载。
+
+## 证书有效期
+
+所有TiDB集群中组件的TLS证书的有效期，可以由您自定义，例如：使用Openssl 签发生成TLS证书时，通过**days**参数设置有效期，详见[生成自签名证书](/generate-self-signed-certificates.md)。
```
Comment (qiancai) [inline enable-tls-between-components.md:198]:
```suggestion
你可以自定义 TiDB 集群中各组件 TLS 证书的有效期。例如，使用 Openssl 签发生成 TLS 证书时，可以通过**days**参数设置有效期，详见[生成自签名证书](/generate-self-signed-certificates.md)。
```

File: enable-tls-between-components.md [hunk 3]
```diff
@@ -189,7 +189,13 @@ aliases: ['/docs-cn/dev/enable-tls-between-components/','/docs-cn/dev/how-to/sec
...
 ## 证书重加载
 
-TiDB、PD 和 TiKV 和各种 Client 都会在每次新建相互通讯的连接时重新读取当前的证书和密钥文件内容，实现证书和密钥的重加载。目前暂不支持 CA 的重加载。
+目前支持TiDB、TiKV、PD的TLS证书，部署数据中心和Cloud上时，自动轮换TLS证书，无需重启TiDB集群；暂不支持TiFlash，TiCDC的自动轮换TLS证书。
+
+TiDB、PD 和 TiKV 和各种 Client 都会在每次新建相互通讯的连接时重新读取当前的证书和密钥文件内容，实现证书和密钥的重加载。
+
+## 证书有效期
+
+你可以自定义 TiDB 集群中各组件 TLS 证书的有效期。例如，使用 Openssl 签发生成 TLS 证书时，可以通过**days**参数设置有效期，详见[生成自签名证书](/generate-self-signed-certificates.md)。
```
Comment (qiancai) [inline enable-tls-between-components.md:198]:
```suggestion
你可以自定义 TiDB 集群中各组件 TLS 证书的有效期。例如，使用 Openssl 签发生成 TLS 证书时，可以通过 **days** 参数设置有效期，详见[生成自签名证书](/generate-self-signed-certificates.md)。
```

File: enable-tls-between-components.md [hunk 4]
```diff
@@ -189,7 +189,13 @@ aliases: ['/docs-cn/dev/enable-tls-between-components/','/docs-cn/dev/how-to/sec
 
 ## 证书重加载
 
-TiDB、PD 和 TiKV 和各种 Client 都会在每次新建相互通讯的连接时重新读取当前的证书和密钥文件内容，实现证书和密钥的重加载。目前暂不支持 CA 的重加载。
+目前支持TiDB、TiKV、PD的TLS证书，部署数据中心和Cloud上时，自动轮换TLS证书，无需重启TiDB集群；暂不支持TiFlash，TiCDC的自动轮换TLS证书。
+
```
[the remark this answers is not in the document]
Comment (lulukelu) [reply enable-tls-between-components.md:193]: 不一样啊，可以改成自己部署在数据中心和自己管理的Cloud上

File: enable-tls-between-components.md [hunk 5]
```diff
@@ -189,7 +189,13 @@ aliases: ['/docs-cn/dev/enable-tls-between-components/','/docs-cn/dev/how-to/sec
 
 ## 证书重加载
 
-TiDB、PD 和 TiKV 和各种 Client 都会在每次新建相互通讯的连接时重新读取当前的证书和密钥文件内容，实现证书和密钥的重加载。目前暂不支持 CA 的重加载。
+目前支持 TiDB、TiKV、PD 的 TLS 证书，部署数据中心和 Cloud 时，支持自动轮换 TLS 证书，无需重启 TiDB 集群。 TiFlash 和 TiCDC 暂不支持自动轮换 TLS 证书。
```
Comment (qiancai) [inline enable-tls-between-components.md:192]:
```suggestion
目前支持重加载 TiDB、TiKV、PD 的 TLS 证书，部署数据中心和 Cloud 时，支持自动轮换 TLS 证书，无需重启 TiDB 集群。 TiFlash 和 TiCDC 暂不支持自动轮换 TLS 证书。
```

File: enable-tls-between-components.md [hunk 6]
```diff
@@ -189,7 +189,13 @@ aliases: ['/docs-cn/dev/enable-tls-between-components/','/docs-cn/dev/how-to/sec
 
 ## 证书重加载
```
Comment (lilin90) [inline enable-tls-between-components.md:190]:
```suggestion
## 证书重新加载
```

**Review by ljun0712 — APPROVED** [review]

File: enable-tls-between-components.md [hunk 7]
```diff
@@ -187,9 +187,18 @@ aliases: ['/docs-cn/dev/enable-tls-between-components/','/docs-cn/dev/how-to/sec
...
-## 证书重加载
+## 证书重新加载
 
-TiDB、PD 和 TiKV 和各种 Client 都会在每次新建相互通讯的连接时重新读取当前的证书和密钥文件内容，实现证书和密钥的重加载。目前暂不支持 CA 的重加载。
+- 如果 TiDB 部署在本地的数据中心，支持重加载 TiDB、TiKV、PD 的 TLS 证书，无需重启 TiDB 集群。
+- 如果 TiDB 部署在自己管理的 Cloud，支持配置自动轮换 TLS 证书，无需重启 TiDB 集群。 
+
+TiFlash 和 TiCDC 暂不支持重加载或自动轮换 TLS 证书。
+
+TiDB、PD 和 TiKV 和各种 Client 都会在每次新建相互通讯的连接时重新读取当前的证书和密钥文件内容，实现证书和密钥的重加载。
```
Comment (lilin90) [inline enable-tls-between-components.md:197]:
```suggestion
- 如果 TiDB 部署在本地的数据中心，支持重新加载 TiDB、TiKV、PD 的 TLS 证书，无需重启 TiDB 集群。
- 如果 TiDB 部署在自己管理的 Cloud，支持配置自动轮换 TLS 证书，无需重启 TiDB 集群。 

TiFlash 和 TiCDC 暂不支持重新加载或自动轮换 TLS 证书。

TiDB、PD、TiKV 和各种 Client 都会在每次新建相互通讯的连接时重新读取当前的证书和密钥文件内容，实现证书和密钥的重新加载。
```

File: enable-tls-between-components.md [hunk 8]
```diff
@@ -187,9 +187,18 @@ aliases: ['/docs-cn/dev/enable-tls-between-components/','/docs-cn/dev/how-to/sec
...
+
+## 证书有效期
+
+你可以自定义 TiDB 集群中各组件 TLS 证书的有效期。例如，使用 Openssl 签发生成 TLS 证书时，可以通过 **days** 参数设置有效期，详见[生成自签名证书](/generate-self-signed-certificates.md)。
```
Comment (lilin90) [inline enable-tls-between-components.md:201]:
```suggestion
你可以自定义 TiDB 集群中各组件 TLS 证书的有效期。例如，使用 OpenSSL 签发生成 TLS 证书时，可以通过 **days** 参数设置有效期，详见[生成自签名证书](/generate-self-signed-certificates.md)。
```

File: enable-tls-between-components.md [hunk 9]
```diff
@@ -187,9 +187,18 @@ aliases: ['/docs-cn/dev/enable-tls-between-components/','/docs-cn/dev/how-to/sec
...
-## 证书重加载
+## 证书重新加载
 
-TiDB、PD 和 TiKV 和各种 Client 都会在每次新建相互通讯的连接时重新读取当前的证书和密钥文件内容，实现证书和密钥的重加载。目前暂不支持 CA 的重加载。
+- 如果 TiDB 部署在本地的数据中心，支持重新加载 TiDB、TiKV、PD 的 TLS 证书，无需重启 TiDB 集群。
+- 如果 TiDB 部署在自己管理的 Cloud，支持配置自动轮换 TLS 证书，无需重启 TiDB 集群。 
+
+TiFlash 和 TiCDC 暂不支持重新加载或自动轮换 TLS 证书。
+
+TiDB、PD、TiKV 和各种 Client 都会在每次新建相互通讯的连接时重新读取当前的证书和密钥文件内容，实现证书和密钥的重新加载。
```
Comment (qiancai) [inline enable-tls-between-components.md:197]:
```suggestion
TiDB、PD、TiKV 和各种 Client 都会在每次新建相互通讯的连接时重新读取当前的证书和密钥文件内容，实现证书和密钥的重新加载。

TiFlash 和 TiCDC 暂不支持重新加载或自动轮换 TLS 证书。
```

File: enable-tls-between-components.md [hunk 10]
```diff
@@ -187,9 +187,18 @@ aliases: ['/docs-cn/dev/enable-tls-between-components/','/docs-cn/dev/how-to/sec
...
     ```
 
-## 证书重加载
+## 证书重新加载
 
-TiDB、PD 和 TiKV 和各种 Client 都会在每次新建相互通讯的连接时重新读取当前的证书和密钥文件内容，实现证书和密钥的重加载。目前暂不支持 CA 的重加载。
+- 如果 TiDB 部署在本地的数据中心，支持重新加载 TiDB、TiKV、PD 的 TLS 证书，无需重启 TiDB 集群。
+- 如果 TiDB 部署在自己管理的 Cloud，支持配置自动轮换 TLS 证书，无需重启 TiDB 集群。 
+
+TiDB、PD、TiKV 和各种 Client 都会在每次新建相互通讯的连接时重新读取当前的证书和密钥文件内容，实现证书和密钥的重新加载。
```
Comment (qiancai) [inline enable-tls-between-components.md:195]:
```suggestion
TiDB、PD、TiKV 和各种 Client 在每次新建相互通讯的连接时都会重新读取当前的证书和密钥文件内容，实现证书和密钥的重新加载。
```

File: enable-tls-between-components.md [hunk 11]
```diff
@@ -187,9 +187,18 @@ aliases: ['/docs-cn/dev/enable-tls-between-components/','/docs-cn/dev/how-to/sec
...
     ```
 
-## 证书重加载
+## 证书重新加载
 
-TiDB、PD 和 TiKV 和各种 Client 都会在每次新建相互通讯的连接时重新读取当前的证书和密钥文件内容，实现证书和密钥的重加载。目前暂不支持 CA 的重加载。
+- 如果 TiDB 部署在本地的数据中心，支持重新加载 TiDB、TiKV、PD 的 TLS 证书，无需重启 TiDB 集群。
+- 如果 TiDB 部署在自己管理的 Cloud，支持配置自动轮换 TLS 证书，无需重启 TiDB 集群。 
+
+TiDB、PD、TiKV 和各种 Client 在每次新建相互通讯的连接时都会重新读取当前的证书和密钥文件内容，实现证书和密钥的重新加载。
```
Comment (qiancai) [inline enable-tls-between-components.md:195]:
```suggestion
- 如果 TiDB 部署在本地的数据中心，TiDB、PD、TiKV 和各种 client 在每次新建相互通讯的连接时都会重新读取当前的证书和密钥文件内容，实现证书和密钥的重新加载，无需重启 TiDB 集群。
- 如果 TiDB 部署在自己管理的 Cloud，支持配置自动轮换 TLS 证书，无需重启 TiDB 集群。 
```

Comment (qiancai) [reply enable-tls-between-components.md:195]: @lulukelu PTAL

File: enable-tls-between-components.md [hunk 12]
```diff
@@ -187,9 +187,14 @@ aliases: ['/docs-cn/dev/enable-tls-between-components/','/docs-cn/dev/how-to/sec
     cert-allowed-cn = ["PD-Server", "TiKV-Server", "TiFlash-Server"]
     ```
 
-## 证书重加载
+## 证书重新加载
 
-TiDB、PD 和 TiKV 和各种 Client 都会在每次新建相互通讯的连接时重新读取当前的证书和密钥文件内容，实现证书和密钥的重加载。目前暂不支持 CA 的重加载。
+- 如果 TiDB 集群部署在本地的数据中心，TiDB、PD、TiKV 和各种 client 在每次新建相互通讯的连接时都会重新读取当前的证书和密钥文件内容，实现证书和密钥的重新加载，无需重启 TiDB 集群。
+- 如果 TiDB 集群部署在自己管理的 Cloud，TiDB、PD、TiKV和TiCDC的组件证书可以支持自动轮换TLS 证书，无需重启TiDB集群，TLS证书签发需要客户与云服务商证书管理服务集成。
```
Comment (qiancai) [inline enable-tls-between-components.md:193]:
```suggestion
- 如果 TiDB 集群部署在自己管理的 Cloud，TiDB、PD、TiKV、TiCDC 组件的 TLS 证书支持自动轮换，无需重启TiDB 集群，TLS 证书签发需要客户与云服务商证书管理服务集成。 
```

**Review by qiancai — APPROVED** [review]

**Pull request merged**